src/frontends/lean/widget.cpp

```diff
@@ -20,7 +20,29 @@ Author: E.W.Ayers
 
 namespace lean {
 
-std::atomic_uint g_fresh_handler_id;
+
```

Comment:
Oh wow, so previously the handler id changed every time a component rerendered?  This explains the "invalid handler" errors.

Reply:
yeah that's right. Needed to change